shuup/core/models/_products.py

```diff
@@ -132,6 +133,17 @@ def __str__(self):
 
 
 class ProductQuerySet(TranslatableQuerySet):
+
+    def _get_visible_modes(self):
+        # TODO: What's the point of this?
+        return [
+                ProductMode.NORMAL,
+                ProductMode.PACKAGE_PARENT,
+                ProductMode.SUBSCRIPTION,
+                ProductMode.SIMPLE_VARIATION_PARENT,
+                ProductMode.VARIABLE_VARIATION_PARENT
+        ]
```

Comment:
So this is all but `VARIATION_CHILD`. Maybe listing the opposite would be easier to understand?